gecoscc/userdb.py

```diff
@@ -13,7 +13,7 @@
 from uuid import uuid4
 import pymongo
 
-from pyramid.security import authenticated_userid
+import pyramid.request
```

Comment:
Do you mean if we could import only a part of `pyramid.request`? Or if it will work without importing `pyramid.request`?